composer.json

```diff
@@ -14,9 +14,14 @@
     "require": {
         "php": ">=5.3.2",
         "twig/twig": "1.18.*",
-        "erusev/parsedown-extra": "dev-master@dev"
+        "erusev/parsedown-extra": "dev-master@dev",
```

Comment:
No, I think that was just overlooked at some point. I would say to use a stable constraint here instead of the dev branch.